ressources.md

```diff
@@ -0,0 +1,15 @@
+# Ressources for Contributing
+
+## Contributing
+
+- [Kubernetes Contributing](https://github.com/kubernetes/community/blob/master/contributors/guide/contributing.md#learn-about-sigs)
+- [Kubernetes Pull-requests](https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md)
+- [Nodejs Contributing](https://github.com/nodejs/node/blob/master/CONTRIBUTING.md)
+
+## Code of Conduct
+
+- [Kubernetes Code of Conduct](https://github.com/kubernetes/community/blob/master/governance.md#code-of-conduct)
+
+## Developer's Certificate of Origin? (Nodejs)
+
+## Rebase over Merge? (Nodejs)
```

Comment:
This all looks still to be WIP?

Is this intended to be changed later? If so, we should probably open up a ticket for it.

Reply:
Good catch! This was never meant to be committed.